drivers/adxl345/adxl345.c

```diff
@@ -43,6 +44,12 @@ int adxl345_init(adxl345_t *dev, const adxl345_params_t* params)
     /* get device descriptor */
     dev->params= *params;
```

Comment:
@haukepetersen How am I suppose to put some members in RAM and others in ROM ? Should I remove them from the struct and call the `#define` params in `adxl345_init` function ?